go/vt/workflow/resharding/horizontal_resharding_workflow.go

```diff
@@ -76,19 +76,21 @@ func (*HorizontalReshardingWorkflowFactory) Init(m *workflow.Manager, w *workflo
 	subFlags := flag.NewFlagSet(horizontalReshardingFactoryName, flag.ContinueOnError)
 	keyspace := subFlags.String("keyspace", "", "Name of keyspace to perform horizontal resharding")
 	vtworkersStr := subFlags.String("vtworkers", "", "A comma-separated list of vtworker addresses")
+	minHealthyRdonlyTablets := subFlags.String("min_healthy_rdonly_tablets", "", "Minimum number of rdonly healthy tablets required")
```

Comment:
Minor nit -- I'd rephrase to "Minimum number of healthy RDONLY tablets required"

Reply:
Also this should use `subFlags.Int` and should be passed through the various layers as an int, not a string.

Reply:
I did as strings because they all get passed down this way:

```
args := []string{splitCmd, "--min_healthy_rdonly_tablets", minHealthyRdonlyTablets, sourceKeyspaceShard}
```

If we do it as an int I have to convert back to string in here. Do you is worth it?

Reply:
Ah yeah -- sorry I noticed this later and forgot to delete my earlier comment. I think it's fine as-is.

Ideally we would want some validation in the UI layer to restrict this to only accept numbers since I think otherwise we'll only find out when the split clone process fails, but that's a pretty minor issue.